README.md

```diff
@@ -105,6 +105,7 @@ It returns a set of functions
 *   `useGlobalState`: a custom hook works like React.useState
 *   `getGlobalState`: a function to get a global state by key outside React
 *   `setGlobalState`: a function to set a global state by key outside React
+*   `subscribe`: a function that subscribes to state changes
```

Comment:
The api doc is generated from the source code. Can you edit jsdoc in code and run `yarn run apidoc`?

Reply:
Done.